cpp/src/arrow/compute/api_vector.h

```diff
@@ -661,6 +661,16 @@ Result<Datum> CumulativeMin(
     const Datum& values, const CumulativeOptions& options = CumulativeOptions::Defaults(),
     ExecContext* ctx = NULLPTR);
 
+/// \brief Compute the cumulative mean of an array-like object
+///
```

Comment:
Can you note that the start_value option is ignored?

Reply:
Or add it in the `CumulativeOptions` docs instead.

Reply:
I added in both